signaturit-sdk.gemspec

```diff
@@ -15,7 +15,11 @@ Gem::Specification.new do |s|
 
     s.license     = 'MIT'
 
-    s.add_dependency 'rest-client', '~> 1.7'
+    s.add_dependency 'rest-client', '> 1.7'
```

Comment:
I think that allow use 2.0 is OK. It is largely compatible with 1.x https://github.com/rest-client/rest-client#upgrading-to-rest-client-20-from-1x